src/tests/ftest/pool/destroy_tests.py

```diff
@@ -232,52 +229,19 @@ def test_destroy_invalid_uuid(self):
         self.validate_pool_creation(hostlist_servers, setid)
 
         # Change the pool uuid
-        valid_uuid = self.pool.pool.uuid
-        self.pool.pool.set_uuid_str("81ef94d7-a59d-4a5e-935b-abfbd12f2105")
+        valid_uuid = self.pool.uuid
+        invalid_uuid = "81ef94d7-a59d-4a5e-935b-abfbd12f2105"
+        self.pool.uuid = invalid_uuid
 
         # Attempt to destroy the pool with an invalid UUID
         self.validate_pool_destroy(
-            hostlist_servers,
-            "with an invalid UUID {}".format(self.pool.pool.get_uuid_str()),
-            True)
-
-        # Restore th valid uuid to allow tearDown() to pass
-        self.log.info(
-            "Restoring the pool's valid uuid: %s", str(valid_uuid.value))
-        self.pool.pool.uuid = valid_uuid
-
-    @skipForTicket("DAOS-5545")
-    def test_destroy_invalid_group(self):
```

Comment:
Why is this test case removed?

Reply:
I explained in the comment.

test_destroy_invalid_group: Removed because we're using dmg and there's no
argument to set group for dmg pool destroy. We used to be able to test with API,
but we can't create a pool with API anymore.

Reply:
I added back in. Now it sets the invalid group name in dmg config file.

Reply:
Could this be added back in the original location in the file to make the diff easier to understand?